src/python/T0/RunConfig/Tier0Config.py

```
@@ -1294,3 +1296,100 @@ def addTier1Skim(config, skimName, dataTier, primaryDataset, cmsswVersion,
 
     datasetConfig.Tier1Skims.append(skimConfig)
     return
+
+
+def addReprocessingRequest(config, inputDataset, globalTag, cmsswVersion,
```

Pylint (4.0.5)

[refactor] Too many positional arguments (7/5)

(R0917)